web/src/Tests/AppRouter.test.js

```diff
@@ -0,0 +1,43 @@
+import { render, screen, cleanup, getByText } from "@testing-library/react";
+import App from "../App";
+import { waitFor } from "@testing-library/react";
+import fetchMock from "jest-fetch-mock";
+import userEvent from '@testing-library/user-event'
+
+beforeEach(() => {
+  fetchMock.enableMocks();
+  fetch.resetMocks();
+});
+
+describe('React Router testing',()=> {
+
+  test('Homepage is rendered ',()=> {
+    render(<App/>)
+    const title = screen.getByText(/HomePage/i)
+    const topicsLink = screen.getByText('Topics')
+    
+  })
+  test('Links are on the HomePage and arent empty string',async ()=> {
+    render(<App/>)
+    const links = screen.getAllByRole('link')
+    expect(links).toBeDefined()
+    expect(links[0]).toBeDefined() //expect there to be at least 1 element in the list
```

Comment:
Maybe a cleaner way to express this would be.
```suggestion
    expect(links.length).toBeGreaterThan(0)
```

Reply:
Nice!